.gitignore

```diff
@@ -11,3 +11,4 @@ build
 .pytest_cache
 .vscode
 .env
+docker-compose.override.yaml
```

Comment:
Fine to keep this, I suppose. But mostly we should be using env file overrides with vars in the compose script, so we get those file updates if they change.